runtimes/dart/versions/latest/helpers/build-flutter.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Safe to ignore this file, separate PR is adding proper flutter implementation. This was done as test for new tests structure